packages/nimble-components/src/wafer-map/modules/computations.module.ts

```diff
@@ -0,0 +1,189 @@
+import { scaleBand, ScaleLinear, scaleLinear } from 'd3-scale';
```

Comment:
We should rename the file to just `computations.ts`
That way it aligns with the class name `Computations` and the test name `computations.spec.ts`

If the name "computations module" is desired then we should do
file name `computations-module.ts` class name `ComputationsModule` and test name `computations-module.spec.ts`

I think including `module` in the name is unnecessary and would prefer `computations.ts`.